extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetialsBody.ts

```diff
@@ -19,33 +19,33 @@ import { SqlMigrationAssessmentResultItem } from '../../service/contracts';
 // Class that defines ui for body section of assessment result page
 export class AssessmentDetailsBody {
 	private _view!: azdata.ModelView;
-	private _model!: MigrationStateModel;
 	private _treeComponent!: TreeComponent;
-	private _instanceSummary = new InstanceSummary();
+	private _instanceSummary;
 	private _databaseSummary = new DatabaseSummary();
 	private _issueSummary = new IssueSummary();
 	private _warningsOrIssuesListSection!: azdata.ListViewComponent;
 	private _findingsSummaryList!: azdata.ListViewComponent;
 	private _disposables: vscode.Disposable[] = [];
 	private _activeIssues!: SqlMigrationAssessmentResultItem[];
 
-	constructor(migrationStateModel: MigrationStateModel) {
-		this._model = migrationStateModel;
+	constructor(public migrationStateModel: MigrationStateModel) {
 		this._treeComponent = new TreeComponent();
+		this._instanceSummary = new InstanceSummary(migrationStateModel);
 	}
 
 	public get treeComponent() {
 		return this._treeComponent;
 	}
 
 	// function that defines all the components for body section
-	public createAssessmentDetailsBody(view: azdata.ModelView): azdata.Component {
+	public async createAssessmentDetailsBody(view: azdata.ModelView): Promise<azdata.Component> {
```

Comment:
Async should be added to the end of the async method name.

Reply:
Changed method name